### PR TITLE
GraphQL: Add coverage for product reviews

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds support for product reviews. 

## Affected DevDocs pages

-  multiple

## Links to Magento source code

-  ...

whatsnew
Added support for product reviews in GraphQL. Use the [createProductReview mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/create-product-review.html) to add a review and the [productReviewRatingsMetadata query](https://devdocs.magento.com/guides/v2.4/graphql/queries/product-review-ratings-metadata.html) to retrieve information about the reviews infrastructure.

### CODE REVIEW

File: src/_includes/graphql/product-review.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+The `ProductReview` object contains details about a product review. It contains the following attributes.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`average_rating` | Float! | The average rating for product review
+`created_at` | String! | Date indicating when the review was created.")
```
Comment (rogyar) [inline src/_includes/graphql/product-review.md:6]: Looks like it was accidentally pasted

```suggestion
`created_at` | String! | Date indicating when the review was created
```

File: src/guides/v2.4/graphql/product/product-interface.md [hunk 2]
```diff
@@ -153,6 +156,19 @@ Attribute |  Data Type | Description
...
 `description` | PriceAdjustmentDescriptionEnum | Indicates whether the entity described by the code attribute is included or excluded from the adjustment.
 
+#### ProductReviews object {#ProductReviews}
+
+`ProductReviews` contains an array of reviews written about the product.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`items` | [ProductReview]! | An array of product reviews
+`page_info` | [SearchResultPageInfo]({{page.baseurl}}/graphql/queries/products.html#SearchResultPageInfo) | Metadata for pagination rendering
```
Comment (rogyar) [inline src/guides/v2.4/graphql/product/product-interface.md:166]:
```suggestion
`page_info` | [SearchResultPageInfo!]({{page.baseurl}}/graphql/queries/products.html#SearchResultPageInfo) | Metadata for pagination rendering
```

File: src/guides/v2.4/graphql/queries/customer.md [hunk 3]
```diff
@@ -501,6 +501,19 @@ Attribute |  Data Type | Description
...
 {% include graphql/customer-orders-output.md %}
 
+#### ProductReviews object {#ProductReviews}
+
+`ProductReviews` contains an array of reviews written about the product.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`items` | [ProductReview]! | An array of product reviews
+`page_info` | [SearchResultPageInfo]({{page.baseurl}}/graphql/queries/products.html#SearchResultPageInfo) | Metadata for pagination rendering
```
Comment (rogyar) [inline src/guides/v2.4/graphql/queries/customer.md:511]:
```suggestion
`page_info` | [SearchResultPageInfo!]({{page.baseurl}}/graphql/queries/products.html#SearchResultPageInfo) | Metadata for pagination rendering
```

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Pull request merged**